packaging/generic-deb/debian/rules

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/make -f
+# -*- makefile -*-
+VERSION := $(shell cat /tmp/ecs-agent-version)
+
+# Uncomment this to turn on verbose mode.
+export DH_VERBOSE=1
+
+%:
+	dh $@
+
+build:
```

Comment:
This target should be named `override_dh_auto_build` rather than just `build`

Reply:
done